3 data.sql

```diff
@@ -0,0 +1,20 @@
+use BicycleShopDb
+go
+delete Shop
+go
+
```

Comment:
The data doesn't run. There are at least 2 issues that prevent it from being allowed into the table. 
Did you run after you finished all constraints on the table?

Reply:
I fixed the issues; it should run now.